src/test/java/MainTest.java

```diff
@@ -9,4 +12,43 @@ public void testSth() {
         assertTrue(true);
     }
 
+
+    /**
+     * Method check setting path
+     * */
+    @Test
+    public void testFileManipulatorSetter() {
```

Comment:
Hah, moje ulubione, bo testy ;)
Do każdej klasy powinna być klasa testowa, czyli jak testujesz klasę WordFinder, to stwórz klasę WordFinderTest w katalogu test (i podkatalogach takich samych jak oryginalna klasa)